src/ensembl/src/shared/components/layout/StandardAppLayout.scss

```diff
@@ -70,6 +70,10 @@ $drawerWindowWidth: 45px;
     );
   }
 
+  &Floating {
+    z-index: 4;
+  }
```

Comment:
This doesn't do anything?

The sidebar still covers the main content area even if I disable the z-index rule.

![image](https://user-images.githubusercontent.com/6834224/75762092-0ca35480-5d32-11ea-9f98-7897452d1782.png)

Reply:
✔️